graphql/types/OrderForm.graphql

```diff
@@ -115,6 +115,12 @@ type OrderFormItem {
   """
   unitMultiplier: Float
   canHaveAttachment: Boolean
+  attachments: [AttachmentItem]
+}
+scalar AttachmentContent
```

Comment:
Why is this a scalar and not a type?

Reply:
Because its format is not known prior

It's similar of what was done here:
https://github.com/vtex-apps/checkout-graphql/blob/master/graphql/types/Item.graphql#L117-L122